scripts/google_health/Google_Health_COVID19_US_StatisticalVariables.mcf

```diff
@@ -0,0 +1,94 @@
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
```

Comment:
All these nodes preexist from the COVID Tracking import (e.g. https://browser.datacommons.org/kg?dcid=CumulativeCount_MedicalConditionIncident_COVID_19_PatientOnVentilator)

However, we are making a schema change right now to change  `medicalCondition: dcs:COVID_19` to `incidentType: dcs:COVID_19` to match the earlier NYT and WHO imports (e.g. https://browser.datacommons.org/kg?dcid=NYTCovid19CumulativeDeaths)

I will handle the reimport required here, but after removing this file, this PR is good to go.